core/ArchiveProcessor/Rules.php

```diff
@@ -343,4 +344,53 @@ public static function shouldProcessSegmentsWhenReArchivingReports()
     {
         return Config::getInstance()->General['rearchive_reports_in_past_exclude_segments'] != 1;
     }
+
+    public static function shouldForceArchiveInvalidatedArchiveInThisRequest(Parameters $params, $doneFlags)
+    {
+        $isArchivePhpTrigerred = !SettingsServer::isArchivePhpTriggered(); // safety to make sure core:archive doesn't trigger archiving if the archive is still usable
```

Comment:
fyi not sure if on purpose...  it says `$isArchivePhpTrigerred` but there is `!` so the variable is maybe the opposite?

Reply:
Mistake, there's another ! below so it works, but I'll remove both.